<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vedo viewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_plane_cutter">
         <property name="text">
          <string>Plane cutter</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_box_cutter">
         <property name="text">
          <string>Box cutter</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_sphere_cutter">
         <property name="text">
          <string>Sphere cutter</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_invert">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Invert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_get_from_napari">
       <property name="text">
        <string>Retrieve mesh from napari</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_send_back">
       <property name="text">
        <string>Send back to napari</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
